src/style.css

```diff
@@ -1,3 +1,81 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
 body {
+  background-color: #000;
```

Comment:
Why did you add a black background to your body?
And you don't need to specify these properties to both `html` and `body` selectors